### ISSUE TITLE
Loading game with selected movement destination in combat

### ISSUE THREAD

**Issue by GoogleCodeExporter** [alistairreilly/andors-trail#306]
```
What steps will reproduce the problem?
1. Engage in combat
2. Long-click a tile to flee to (but don't flee)
3. Save the game (or exit to title screen)
4. Load the savegame
5. Notice how the combat bar is incorrectly populated. For example, the attack 
button displays "Attack (%1$d AP)" instead of "Move (6AP)", and the monster HP 
bar is visible even though no monster is selected.

Confirmed on v0.6.11 on Google Play.

```

Original issue reported on code.google.com by `oskar.wi...@gmail.com` on 12 Jun 2012 at 8:31

**Comment by GoogleCodeExporter** [comment]
```
Fixed in da6aacde5adb .
```

Original comment by `oskar.wi...@gmail.com` on 16 Jun 2012 at 10:21
- Changed state: **Ready**

**Comment by GoogleCodeExporter** [comment]
```
Released in v0.6.12.
```

Original comment by `oskar.wi...@gmail.com` on 28 Oct 2012 at 7:43
- Changed state: **Fixed**
- Added labels: **Milestone-0.6.12**
- Removed labels: **Milestone-0.6.11**
